### ISSUE TITLE
Video Not Getting Scraped from Messages

### ISSUE THREAD

**Issue by drvnd** [sim0n00ps/OF-DL#305]
I'm trying to download a video sent in messages from a model and OF-DL seems to be skipping the message. Initially it was sent in a bundle of pictures but I had them reupload it separately and OF-DL still skips it. The pictures are getting scraped without any issues. 

The only thing of note is the the model has DRM turned on and the video is DRM'ed. 

There is also the following message that happens several times after selecting the account from Custom but it does proceed and download afterwards:

Exception caught: Response status code does not indicate success: 403 (Forbidden).

StackTrace:    at System.Net.Http.HttpResponseMessage.EnsureSuccessStatusCode()
   at OF_DL.Helpers.APIHelper.GetDRMMPDPSSH(String mpdUrl, String policy, String signature, String kvp, Auth auth)

Downloading 14 Messages ----------------------------------------  92%

Messages Already Downloaded: 13 New Messages Downloaded: 0
Getting Paid Messages
Found 0 Paid Messages

It just will not download that last message for whatever reason. Thank you for this tool and all of your hard work!

**Comment by sim0n00ps** [comment]
Try using a VPN if you haven't already done so. Also delete the metadata folder for that user and try scraping again.

**Comment by drvnd** [comment]
No luck. I deleted the metadata folder. It finds 14 message but only downloads 13.

Downloading 14 Messages ----------------------------------------  92%

Messages Already Downloaded: 13 New Messages Downloaded: 0

I tried a VPN connection after and same issue. I wonder if OF changed something. I'm not used to seeing these errors:

2024-03-10 00:29:41.236 -06:00 [ERR] Exception caught: Response status code does not indicate success: 403 (Forbidden).

StackTrace:    at System.Net.Http.HttpResponseMessage.EnsureSuccessStatusCode()
   at OF_DL.Helpers.APIHelper.GetDRMMPDPSSH(String mpdUrl, String policy, String signature, String kvp, Auth auth)
2024-03-10 00:29:41.396 -06:00 [ERR] Exception caught: Response status code does not indicate success: 403 (Forbidden).
